torchinductor/sizevars.py

```diff
@@ -83,8 +85,14 @@ def remove_zero_terms(base, divisor):
         def visit_indexing_div(base, divisor):
             return IndexingDiv(remove_zero_terms(base, divisor), divisor)
 
-        def visit_moduler_indexing(base, divisor, modulus):
+        def visit_modular_indexing(base, divisor, modulus):
             base = remove_zero_terms(base, divisor)
+            if isinstance(base, sympy.Expr):
```

Comment:
```suggestion
            if isinstance(base, (sympy.Expr, sympy.Symbol)):
```
I think with this change we should be able to delete lines 96 through 102.

Reply:
Actually, we might be able to just remove this condition entirely and run this always?  I think the only other case is a `sympy.Integer`, and this should work for that fine too.